internal/operators/ocs/validations.go

```diff
@@ -110,6 +110,20 @@ func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
 		return false
 	} else {
 		for _, host := range hosts { // if the worker nodes >=3 , install OCS on all the worker nodes if they satisfy OCS requirements
+
+			// if number of hosts are >=6 and one of them has Role as Auto-assign
```

Comment:
Can the comment elaborate on why this is a limitation?

Reply:
Done